filebeat/config/config.go

```diff
@@ -44,32 +44,32 @@ type FilebeatConfig struct {
 }
 
 type ProspectorConfig struct {
-	Paths                 []string
+	ExcludeFiles          []string `yaml:"exclude_files"`
+	ExcludeFilesRegexp    []*regexp.Regexp
```

Comment:
so we've got a mix of fields filled via config file and fields done programatically? Hmmm....